clients/linux/src/backend.rs

```diff
@@ -233,6 +234,35 @@ impl LbCore {
         ))
     }
 
+    pub fn import_file(
+        &self,
+        parent: Uuid,
+        destination: String,
+        f: Option<Box<dyn Fn(ImportExportFileInfo)>>,
+    ) -> LbResult<()> {
+        import_file(&self.config, parent, PathBuf::from(destination), f).map_err(map_core_err!(ImportFileError,
+            FileAlreadyExists => uerr_dialog!("A file already exists in the path with the same name."),
```

Comment:
```suggestion
            FileAlreadyExists => uerr_dialog!("File collision detected during import."),
```

Reply:
This seems like *the* error people may get, and we should probably provide a better error for this in the future. Can you make an issue for this?

Reply:
Sure, [here](https://github.com/lockbook/lockbook/issues/849) it is.